package/yast2-s390.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-s390
-Version:        4.0.5
+Version:        4.1.0
```

Comment:
Great, we can mark it in https://trello.com/c/U8TfoGXL/2831-sles15-sp1-p2-1124009-bump-yast-package-versions-to-41x as done. :wink:

Reply:
Done.